### DISCUSSION TITLE
Fotos de contatos

### DISCUSSION THREAD

**Discussion by brendewguedes** [chatwoot/chatwoot#7288] [Self hosted]
Fotos de contatos não aparecem

**Comment by vishnu-narayanan** [comment]
1. Could you share more details about your installation setup? 
2. Which version of Chatwoot are you on? 
3. Have you configured a cloud storage provider?

**Comment by tiictechnology** [comment]
@brendewguedes você conseguiu resolver seu problema?
Estou usando o Chatwoot versão 3.7.0 e todos os contatos não tem foto;

Fiz a instalação em uma VPS usando os comandos:
wget https://get.chatwoot.app/linux/install.sh
chmod +x install.sh

Configurei o WhatsApp Cloud Api ((https://www.chatwoot.com/docs/product/channels/whatsapp/whatsapp-cloud/))
![Captura de tela 2024-03-21 164638](https://github.com/chatwoot/chatwoot/assets/19560992/8278bbed-071f-443c-9ef5-0a639a4f80e7)

Desde já muito obrigado pela atenção.

**Reply by sojan-official** [reply]
Whatsapp doesn't provide profile pictures of the contacts: 
ref: https://stackoverflow.com/questions/43390779/how-to-get-whatsapp-profile-picture-using-contact-number

**Reply by tiictechnology** [reply]
Eu até imaginei que poderia ser algo sobre solicitação de análise do aplicativo para aprovação do Facebook/Instagram no uso.

Referencia: https://docs.tactium.com.br/base/solicitando-analise-do-aplicativo-para-aprovacao-do-facebook-no-uso-do-tactium-omni/
